.github/workflows/cross-compile.yml

```diff
@@ -4,14 +4,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go: [ "1.18.x", "1.19.x" ]
+        go: [ "1.18.x", "1.19.x", "1.20.0-rc.1" ]
```

Comment:
hate to be that guy but 1.20.0-rc.2 got released today :)

Reply:
Yes, I'll update this before the PR gets merged.

This PR is currently (soft-) blocked on https://github.com/lucas-clemente/quic-go/pull/3636 though.

Reply:
Updated the PR. Fortunately, there were no changes to crypto/tls since rc1, so we don't have to update qtls.